lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -891,8 +891,7 @@ def get_proj(self):
                                                 self._focal_length)
 
         # Combine all the transformation matrices to get the final projection
-        M0 = np.dot(viewM, worldM)
-        M = np.dot(projM, M0)
+        M = np.linalg.multi_dot([projM, viewM, worldM])
```

Comment:
I think multi_dot does `np.dot(np.dot(projM, viewM), worldM)` whereas the original code does `np.dot(projM, np.dot(viewM, worldM))`.    If the order of rotation matters, those will be different.